components/callout.js

```diff
@@ -35,12 +42,16 @@ const Callout = {
   computed: {
     classes() {
       return {
-        'c-callout--error': this.type === 'error',
-        'c-callout--success': this.type === 'success',
-        'c-callout--warning': this.type === 'warning',
-        'c-callout--info': this.type === 'info'
+        'c-callout--dialog': this.dialog,
+        [`c-callout--${this.type}`]: this.type
       };
     }
+  },
+  methods: {
+    close() {
+      this.$destroy();
+      this.$el.parentNode.removeChild(this.$el);
```

Comment:
Is there a better way to remove this element? And how does this impact `v-if`, what if the call-out should appear again?

Reply:
We should talk about this, im not certain this is the best way, but it some kind of a way